eng/pipelines/release-build-pipeline.yml

```diff
@@ -65,230 +65,249 @@ variables:
   # Import config group. This may direct the build to use secrets from the other groups.
   - group: ${{ parameters.goReleaseConfigVariableGroup }}
 
-jobs:
-  - template: jobs/releasego.yml
-    parameters:
-      releaseVersion: ${{ parameters.releaseVersion }}
-      releaseIssue: ${{ parameters.releaseIssue }}
-      retryInstructionsFlags: '-checkboxes'
-      variables:
-        # Set a variable for each polling parameter. If a variable starts off with 'nil', it will be
-        # overridden with a logging command as the pipeline moves on.
-        poll1MicrosoftGoPRNumber: ${{ parameters.poll1MicrosoftGoPRNumber }}
-        poll2MicrosoftGoCommitHash: ${{ parameters.poll2MicrosoftGoCommitHash }}
-        poll3MicrosoftGoBuildID: ${{ parameters.poll3MicrosoftGoBuildID }}
-        poll4MicrosoftGoImagesPRNumber: ${{ parameters.poll4MicrosoftGoImagesPRNumber }}
-      steps:
-        # Resume the build at the furthest step based on the inputs. A fresh build has all 'nil', a
-        # retry build will have a non-nil value in one of these parameters. Only the value of the last
-        # step's parameter matters.
-        - ${{ if eq(parameters.poll4MicrosoftGoImagesPRNumber, 'nil') }}:
-          - ${{ if eq(parameters.poll3MicrosoftGoBuildID, 'nil') }}:
-            - ${{ if eq(parameters.poll2MicrosoftGoCommitHash, 'nil') }}:
-              - ${{ if eq(parameters.poll1MicrosoftGoPRNumber, 'nil') }}:
-                # Poll for the right commit to use for this release.
+resources:
+  repositories:
+    - repository: 1ESPipelineTemplates
+      type: git
+      name: 1ESPipelineTemplates/1ESPipelineTemplates
+      ref: refs/tags/release
+
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
+  parameters:
+    sdl:
+      sourceAnalysisPool:
+        name: NetCore1ESPool-Svc-Internal
+        image: 1es-windows-2022
+        os: windows
+      tsa:
+        enabled: true
+        configFile: $(Build.SourcesDirectory)/eng/compliance/tsaoptions.json
+
+    stages:
+      - template: stages/releasego.yml
```

Comment:
For the sake of minimizing changes in this PR, what about keeping `releasego.yml` unmodified and define the stage in here instead?

Something like this:

```yml
extends:
  template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
  parameters:
    ...
    stages:
    - stage: Release
       jobs:
       - template: jobs/releasego.yml
```

Reply:
> - template: jobs/releasego.yml

In case it comes up later: this is the case where you need to use `- template: /eng/pipelines/jobs/releasego.yml@self`, because a relative string here is evaluated relative to the `v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates` that's being extended.

Reply:
Good call, the repetition in each pipeline by declaring the stage is quite minimal, and the diff is much clearer.